exporter/prometheusremotewriteexporter/config.go

```diff
@@ -118,5 +122,5 @@ func (cfg *Config) Validate() error {
 		return errors.New("compression type must be snappy")
 	}
 
-	return nil
+	return cfg.RemoteWriteProtoMsg.Validate()
```

Comment:
I tried starting the component using `protobuf_message: io.prometheus.write.v2.Request` without setting the feature flag. I got no errors during startup.

I assume we're currently silently using PRWv1 when the feature flag isn't enabled. Could we error the config validation if the feature flag isn't enabled? Or at the bare minimum log that we're switching to v1. I'd prefer a startup failure though 😬

Reply:
Your assumption is right you can see that here https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/35888/files#diff-2631d8a8855fdc08cf077c0ab99544147ca73eec572bc871d8fe717e826eda38R243

I added the logic you wanted in the last commit.